Add a Quick Settings Tile to control/monitor service

edgarogh opened the pull request:
Adds a Quick Settings Tile to quickly toggle the service state and monitor its state. Most context can be found in #135.

A lot of things in the PR may be imperfect. Feel free to ask for changes.

<p align="center">
<img src="https://github.com/slowscript/warpinator-android/assets/46636609/f4261be8-bce0-4c63-8fcb-3396983d7041" width="50%">
</p>

slowscript reviewed (commented):
Thanks for the PR! This is a nice feature. Code looks good to me.
Just some minor possible improvements, I might even do it myself if I get to this again soon.

edgarogh commented:
Oh I didn't get notified of your comment, I'll fix that asap
Glad that you like it 😉

edgarogh commented on app/src/main/AndroidManifest.xml:
```diff
@@ -84,6 +87,20 @@
+        <service android:name=".TileMainService"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:icon="@drawable/ic_launcher_foreground"
```
Definitely, I noticed the rather small size of the quick tile but didn't realize there was already a bigger icon in the codebase

edgarogh commented:
So I fixed the icon size, filter to only display connected devices and I updated the strings to "X connected" instead of "X in range" (EN and FR). I merged `master` into my branch but you'll probably rebase/squash anyway.

slowscript commented:
Thanks. The string saying "in range" was probably fine, but I will leave this up to the translators.
